lib/single_cov.rb

```diff
@@ -76,6 +76,26 @@ def assert_tested(files: glob('{app,lib}/**/*.rb'), tests: default_tests, untest
       end
     end
 
+    def assert_full_coverage(tests: default_tests, currently_complete: [], currently_complete_file: "")
+      complete = tests.select { |file| File.read(file) =~ /SingleCov.covered!(\s*|\s*\#.*)$/ }
+      missing_complete = currently_complete - complete
+      newly_complete = complete - currently_complete
+      errors = []
+
+      unless missing_complete.empty?
+        errors << "The following file(s) were marked as 100% SingleCov test coverage (had no `coverage:` option) and are no longer. " +
+        "Please extend test coverage in these files to maintain 100% coverage. #{missing_complete}"
+      end
+
+      unless newly_complete.empty?
+        errors << "The following files are newly at 100% SingleCov test coverage. " +
+        "Please add the following to #{currently_complete_file} to ensure 100% coverage is maintained moving forward. " +
+        "#{newly_complete}"
+      end
+
+      raise errors.join("\n") unless errors.empty?
```

Comment:
```suggestion
      assert errors.empty?, errors.join("\n")
```

Reply:
When I make this change, I get the following error message.  No other methods in this file leverage "assert".  All leverage "raise".  

What needs to happen to make the suggested `assert errors.empty?, errors.join("\n")` work without prompting this error?

Alternatively, would you prefer to go with `raise errors.join("\n") unless errors.empty?` given the error?

Error:

```
Cleanliness#test_0036_ensures tests with 100% coverage stay at 100%:
NoMethodError: undefined method `assert' for SingleCov:Module
Did you mean?  assert_used
    /Users/alison.acuna/Code/single_cov/lib/single_cov.rb:98:in `assert_full_coverage'
    ./test/integration/cleanliness_test.rb:448:in `block (2 levels) in <main>'"
```

Reply:
ah you are right ... I was thinking we had the testing framework available, raise is fine 👍

Reply:
Ty <3